Client/Frontend/Browser/LinkPreviewViewController.swift

```diff
@@ -24,7 +24,7 @@ class LinkPreviewViewController: UIViewController {
       $0.setURLSchemeHandler(InternalSchemeHandler(), forURLScheme: InternalURL.scheme)
     }
 
-    let wk = WKWebView(frame: view.frame, configuration: configuration)
+    let wk = BraveWebView(frame: view.frame, configuration: configuration)
```

Comment:
Does this break previews for sites that save cookies for things? Creating a `BraveWebView` like this is actually creating a web view with a non-persistent store (third arg is `isPrivate: Bool = true`), but just wanted to make sure that a preview can still view authorized content despite it being non-persistent

Reply:
Good catch, I will double check.
I think we will want to harden this code even more, inject our scripts to it

Reply:
We may want to actually pass in `isPrivate: PrivateBrowsingManager.shared.isPrivateBrowsing`